token/client.go

```diff
@@ -26,32 +27,15 @@ func New(params *stripe.TokenParams) (*stripe.Token, error) {
 }
 
 func (c Client) New(params *stripe.TokenParams) (*stripe.Token, error) {
-	body := &stripe.RequestValues{}
-	token := c.Key
-
-	if len(params.Customer) > 0 {
-		body.Add("customer", params.Customer)
-	}
-
-	if params.Card != nil {
-		params.Card.AppendDetails(body, true)
-	} else if params.Bank != nil {
-		params.Bank.AppendDetails(body)
-	} else if params.PII != nil {
-		params.PII.AppendDetails(body)
-	} else if len(params.Customer) == 0 {
-		err := errors.New("Invalid Token params: either Card, Bank, or PII need to be set")
-		return nil, err
-	}
-
-	if len(params.Email) > 0 {
-		body.Add("email", params.Email)
+	if params.Bank == nil && params.Card == nil && params.PII == nil {
```

Comment:
I think there's a mistake here because the original if/else also handled `if len(params.Customer)`. Might be easier to just remove the test and let the API error out.

Reply:
Yeah. Done in cb5886a.